elf_reader.go

```diff
@@ -1259,11 +1262,19 @@ func (ec *elfCode) loadDataSections() error {
 				continue
 			}
 
+			if off+sym.Size > uint64(len(data)) {
```

Comment:
Have you run into these cases in practice?

Reply:
Nope, just defense in depth to return an error instead of panicking.